runner/cf-driver/cleanup.sh

```diff
@@ -10,6 +10,27 @@ set -eo pipefail
 # trap any error, and mark it as a system failure.
 trap 'exit $SYSTEM_FAILURE_EXIT_CODE' ERR
 
+cleanup_services () {
+    container_id_base="$1"
+    ci_job_services="$2"
+
+    if [ -z "$ci_job_services" ]; then
+       echo "No services defined in ci_job_services - Skipping service cleanup"
+       return
+    fi
+
+    for a in $(echo "$ci_job_services" | jq -rc '.[] | select(.alias != null).alias'); do
+        container_id="${container_id_base}-svc-${a}"
+
+        cf delete -f "$container_id"
```

Comment:
[suggestion] adding `-r` will remove any routes at the same time.

Reply:
And so I have!